src/main/java/de/terrestris/shogun/model/MapLayer.java

```diff
@@ -456,10 +445,11 @@ public void setMetadata(Set<LayerMetadata> metadata) {
 	/**
 	 * @return the groups
 	 */
-	@ManyToMany(mappedBy="mapLayers", fetch=FetchType.EAGER)
+	@ManyToMany(mappedBy="mapLayers", fetch=FetchType.LAZY)
 //	@JsonIgnore
```

Comment:
```suggestion
```

Reply:
solved in https://github.com/terrestris/shogun-legacy/pull/70/commits/dfacf681b632a11b274e324d06efa5102ed97afc